package.json

```diff
@@ -36,7 +42,8 @@
     "vue-template-compiler": "^2.6.11"
   },
   "gitHooks": {
-    "pre-commit": "lint-staged"
+    "pre-commit": "lint-staged",
+    "pre-push": "npm test"
```

Comment:
Just a suggestion ... some like it, some not 😬

Reply:
I like it a lot :) Skipped it for the backend because I was unsure how to setup tests for the CDK stuff. But now as we also have some first tests there, it be also for the backend a very good addition.